lib/decimal.ex

```diff
@@ -1014,6 +1014,20 @@ defmodule Decimal do
     round(decimal(num), n, mode)
   end
 
+  @doc """
+  Returns the sum of all decimals in the enumerable.
+
+  ## Examples
+
+      iex> Decimal.sum([Decimal.new("1.234"), Decimal.new(2)])
+      Decimal.new("3.234")
+
+  """
+  @spec sum([decimal]) :: t
+  def sum(decimals) do
+    Enum.reduce(decimals, Decimal.zero(), fn d, acc -> Decimal.add(d, acc) end)
```

Comment:
For whatever it's worth, if you don't mind the shorthand lambda syntax, you could condense `sum` to this:
```suggestion
    Enum.reduce(decimals, Decimal.zero(), &Decimal.add/2)
```

Reply:
Hey @J3RN,

I don't mind the lambda syntax at all, I'm just not used to see it for an arity > 1, this is completely fine though.